src/org/usfirst/frc2175/command/group/RunIntakeInGroup.java

```diff
@@ -12,13 +12,15 @@
  */
 public class RunIntakeInGroup extends CommandGroup {
 
+    int speed = sideBeltSpeedForward.getSideBeltSpeedForward();
+
     public RunIntakeInGroup(DriverStation driverStation,
             RobotSubsystems robotSubsystems) {
```

Comment:
To really do this right, you need to modify this constructor so it takes an instance of `IntakeConfig`. Then at the start of the constructor you can pull out the speed values you want.